modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/ItemSelectorImpl.java

```diff
@@ -265,6 +260,33 @@ public PortletURL getItemSelectorURL(
 
 	@Activate
 	protected void activate(BundleContext bundleContext) {
+		_itemSelectorCriterionHandlerServiceTrackerMap =
+			ServiceTrackerMapFactory.openSingleValueMap(
+				bundleContext,
+				(Class<ItemSelectorCriterionHandler<ItemSelectorCriterion>>)
+					(Class<?>)ItemSelectorCriterionHandler.class,
+				null,
+				(serviceReference, emitter) -> {
```

Comment:
@W2jx Since you don't need to get any property from the  `serviceReference`, you should be able to use `com.liferay.osgi.service.tracker.collections.map.ServiceReferenceMapperFactory#create` to simplify this.

Reply:
I will fix it.